### PR TITLE
Documenting Enzyme configuration with Karma

### LINKED ISSUE #1257: Example of configuring Enzyme Adapter with Karma
The install instructions detail adding the adapter configuration to your 'setup file'. I can't figure out what the right incantations are for running via Karma. Is there an up to date example of combining Enzyme + Karma with adapter configuration?

### PR DESCRIPTION
**Pull request by GreenGremlin**
@ljharb asked me to update the Karma documentation with instructions on configuring Enzyme using an Enzyme setup file.

Closes #1257.

### CODE REVIEW

**Comment by GreenGremlin** [comment]
Not completely sure, but this method likely covers more than just Karma. I'm pretty sure it covers mocha-webpack, but I'm not familiar enough to know for sure.

File: docs/guides/karma.md [hunk 1]
```diff
@@ -1,55 +1,61 @@
...
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import jasmineEnzyme from 'jasmine-enzyme';
+
+// Configure Enzyme for the appropriate React adapter
+Enzyme.configure({adapter: new Adapter()});
+
+// Initialize global helpers
+beforeEach(() => {
+    jasmineEnzyme();
```
Comment (ljharb) [inline docs/guides/karma.md]: nit: 2 spaces

File: docs/guides/karma.md [hunk 2]
```diff
@@ -1,55 +1,61 @@
...
 
-See the [webpack guide](webpack.md).
+```js
+/* test/enzyme.js */
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import jasmineEnzyme from 'jasmine-enzyme';
+
+// Configure Enzyme for the appropriate React adapter
+Enzyme.configure({adapter: new Adapter()});
```
Comment (ljharb) [inline docs/guides/karma.md]: nit: spaces inside the curly braces

File: docs/guides/karma.md [hunk 3]
```diff
@@ -1,55 +1,61 @@
...
-        ['babelify', { presets: ['airbnb'] }],
-      ],
-    },
-    // ...
-  });
-};
+/* test/index_test.js */
+import './enzyme';
+
+const testsContext = require.context(".", true, /_test$/)
```
Comment (ljharb) [inline docs/guides/karma.md]: nit: single quotes, semicolon

File: docs/guides/karma.md [hunk 4]
```diff
@@ -1,55 +1,61 @@
...
+
+testsContext.keys().forEach(testsContext)
```
Comment (ljharb) [inline docs/guides/karma.md]: nit: semicolon

File: docs/guides/karma.md [hunk 5]
```diff
@@ -1,55 +1,61 @@
...
-          },
-        }],
-      },
-    },
-    // ...
-  });
-};
+/* some_test.js */
+// Import anything you would normally import `from 'enzyme'` from the Enzyme setup file
+import {shallow} from './test/enzyme';
```
Comment (ljharb) [inline docs/guides/karma.md]: nit: spaces inside the curly braces

File: docs/guides/karma.md [hunk 6]
```diff
@@ -1,55 +1,61 @@
...
+
+const testsContext = require.context(".", true, /_test$/)
+
+testsContext.keys().forEach(testsContext)
+```
+
+```js
+/* some_test.js */
+// If Enzyme is configured in the test entry file, Enzyme can be imported directly
+import {shallow} from 'enzyme';
```
Comment (ljharb) [inline docs/guides/karma.md]: nit: spaces inside the curly braces

**Comment by GreenGremlin** [comment]
> I'll fix all the style nits as part of merging, if you don't beat me to it.

Too late, I already got them.

**Pull request merged**